x/datadeal/types/message_deal.go

```diff
@@ -67,10 +72,6 @@ func (m *MsgSubmitConsent) ValidateBasic() error {
 		return sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "consent is empty")
 	}
 
-	if m.Consent.Certificate == nil {
-		return sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "certificate is empty")
-	}
-
```

Comment:
This code block has been moved into the `Conset.ValidateBasic()`.
https://github.com/medibloc/panacea-core/pull/618/files#diff-96ca4563a066ba12d7027c1dfae0f3379db834ade078b47a86b8bd6000ace671R9